docs/features/experiment.md

```diff
@@ -57,6 +57,11 @@ spec:
     service:
       # Name of the Service (optional). If omitted, service: {} would also be acceptable.
       name: service-name
+      # Service port mappings(optional). This is useful when the container port is different from the desired service port.
+      # If omitted, the port will be mapped to the container port.
+      portMappings:
```

Comment:
why not just go with `ports` to keep it consistent with how we write a service in k8s?

Reply:
I agree with this, we need to be forward thinking in possibly embeding the whole service spec and to do that we would want to match it's api.

Reply:
https://github.com/argoproj/argo-rollouts/issues/2233#issuecomment-1287110944

Reply:
thanks for the review guys. I'll have a look asap.

Reply:
@meeech @zachaller done!  I amended my previous commits.